presidio-analyzer/conf/analyzer.yaml

```diff
@@ -0,0 +1,49 @@
+recognizer_registry:
+  recognizers: 
+    - 
+      name: CreditCardRecognizer
+      supported_language: 
+      - en
+      - es
```

Comment:
A recognizer can only support one language. If we want `CreditCardRecognizer` to support multiple, we need to have multiple instances of it. The reason is that context words differ between languages.

Reply:
If the intention was to allow multiple CreditCardRecognizers, I would suggest to split it into two objects and provide different context words to each. WDYT?